README.md

```diff
@@ -2,6 +2,10 @@
 
 # Kyma CLI
 
+> [!WARNING]
```

Comment:
It's based on [this](https://github.com/orgs/community/discussions/16925) article